core/src/main/java/com/sequenceiq/cloudbreak/init/clusterdefinition/DefaulClusterDefinitionCache.java

```diff
@@ -76,7 +79,12 @@ public void loadBlueprintsFromFile() {
     }
 
     private List<String> blueprints() {
-        return Stream.concat(releasedBlueprints.stream().filter(StringUtils::isNoneBlank),
-                internalBlueprints.stream().filter(StringUtils::isNoneBlank)).collect(Collectors.toList());
+        return Stream.concat(
```

Comment:
Is it possible to concat first and then filter in one step?

Reply:
good catch. fixed